webpack.config.js

```diff
@@ -36,12 +36,26 @@ module.exports = (env, argv) => {
     performance: { hints: false },
     module: {
       rules: [
+        {
+          test: /\.worker\.ts$/,
+          use: [
+            {
+              loader: 'worker-loader',
+              options: {
+                filename: '[name].[contenthash].worker.js'
+              }
+            },
+            {
+              loader: 'ts-loader'
+            }
+          ]
+        },
         {
           test: /\.tsx?$/,
           loader: 'ts-loader'
         },
         {
-          test: /\.css$/,
+          test: /\.(css|scss)$/,
```

Comment:
I don't think adding this extension here is going to work without also adding `sass` and `sass-loader` as dependencies and configuring them in this file, which we may want to do at some point, but I don't think it makes sense to just add the extension.

Reply:
I added `sass` and `sass-loader` to the project, and converted `lava-coder-view.css` to `.scss`. Hopefully I did everything correctly, but please double check because I'm not super comfortable in this space.

Reply:
LGTM other than having the sass dependencies in `devDependencies` rather than `dependencies` in `package.json`.